ddr/ddr/Models/GameEntities/TutorialImage.swift

```diff
@@ -17,7 +17,7 @@ class TutorialImage: SKSpriteNode {
   let rotationDuration = 0.5
   let waitDuration = 0.5
   
-  init(third: Int, rotateRight: Bool) {
+  init(third: Int, rotateRight: Bool = false, alternate: Bool = false) {
```

Comment:
Okay so this class definition is really janky because of the dual constraints of rotation direction and alternation.

I wanted to initialize with a state enum but couldn't figure it out in a timely manner. @mjkern any ideas would be welcome but it works for now